src/Components/Shared/DateFormat.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const format = (dateString) => {
+    const options = { year: 'numeric', month: 'long', day: 'numeric' };
+    return new Date(dateString).toLocaleDateString('en-US', options);
+};
+
+const DateFormat = ({dateString}) => {
+    return <time dateTime={dateString}>{format(dateString)}</time>;
```

Comment:
Are you sure that the `dateTime` value is formatted as the html expect? Is ant of the format described here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time?

Reply:
![image](https://user-images.githubusercontent.com/37185433/52145213-02da8980-2626-11e9-9810-826197e92e00.png)